pinot-common/src/main/java/org/apache/pinot/common/request/AggregationInfo.java

```diff
@@ -30,7 +30,7 @@
  *  Aggregation
  *
  */
-@javax.annotation.Generated(value = "Autogenerated by Thrift Compiler (0.13.0)", date = "2021-09-28")
+@javax.annotation.Generated(value = "Autogenerated by Thrift Compiler (0.13.0)", date = "2022-03-28")
```

Comment:
Since we have upgraded thrift to `0.15.0`, consider re-compiling the files with `0.15.0`

Reply:
hmm, maybe do it in another PR for thrift upgrade?

Reply:
I will submit the new PR after this.